code/core/system/proxy-fh16/include/CANMessageDataStore.h

```diff
@@ -47,11 +47,12 @@ namespace proxy {
  */
 class CanMessageDataStore : public automotive::odcantools::MessageToCANDataStore {
    public:
-    CanMessageDataStore(std::shared_ptr< automotive::odcantools::CANDevice > canDevice);
+    CanMessageDataStore(std::shared_ptr< automotive::odcantools::CANDevice > canDevice, bool readOnly = false);
```

Comment:
My idea was to have read/write mode by default, as it -practically- was so far. 
If this is not the case I may need to also change the nature of the proxy-fh16.readOnlyMode configuration parameter, from an optional parameter to a mandatory one.